src/conn/conn_log.c

```diff
@@ -440,9 +440,9 @@ __log_file_server(void *arg)
 				 * this worker thread process that older file
 				 * immediately.
 				 */
-				if ((log->sync_lsn.file <
-				    log->bg_sync_lsn.file) ||
-				    (log->sync_lsn.file < min_lsn.file))
+				if ((log->sync_lsn.l.file <
+				    log->bg_sync_lsn.l.file) ||
+				    (log->sync_lsn.l.file < min_lsn.l.file))
 					continue;
```

Comment:
Hi. I'm researching on wiredtiger source recently. May I ask a question ? After the `continue`, If I assert `log->sync_lsn.l.file == log->bg_sync_lsn.l.file` and `log->sync_lsn.l.file == min_lsn.l.file`, is that right ? Because sync lsn should not exceed write lsn. (`sync` occurs after `write`).

Reply:
Hi @baiwfg2, this pull request had been merged for a really long time. If you want better attention of your question, I'd suggest raising a topic in the [wiredtiger-users](https://groups.google.com/g/wiredtiger-users) google group.